include/capstone/ppc.h

```diff
@@ -1194,6 +1195,43 @@ typedef enum ppc_insn {
 	PPC_INS_POPCNTD,
 	PPC_INS_POPCNTW,
 	PPC_INS_PTESYNC,
+	PPC_INS_PSQ_L,
```

Comment:
consider adding PS instructions to the end of the enum to avoid breaking changes in opcode IDs